packages/mediapipe-core/lib/src/io/task_options.dart

```diff
@@ -209,3 +209,30 @@ class ClassifierOptions extends BaseClassifierOptions
     }
   }
 }
+
+/// {@macro EmbedderOptions}
+class EmbedderOptions extends BaseEmbedderOptions
+    with InnerTaskOptions<bindings.EmbedderOptions> {
+  /// {@macro EmbedderOptions}
+  const EmbedderOptions({
+    this.l2Normalize = false,
+    this.quantize = false,
+  });
```

Comment:
Coming back around, I think I see now that bools are defined for this in the generated bindings. Can we handle the bools internally by having the user configure with the enum? This would avoid user error using the bools directly and having both false, or both true.

Reply:
This confused me for a long time, too. The two fields are actually unrelated. They can both be true or both be false.

Reply:
Oh wow! What happens in those cases?
![image](https://github.com/google/flutter-mediapipe/assets/16964204/d0ba5516-6221-4876-a8ee-dfa12b51b1f7)

Reply:
The `quantize` parameter determines whether the Embedder emits Embeddings of type `quantized` (integers) or `float` (... floats, obvi). The field name and parameter comes from low level C structs and protos, but I've captured this distinction in the `EmbeddingType` enum which appears on each `Embedding` object.

Separately, `l2Normalize` controls whether the resulting numbers are normalized in the "l2" way - whatever that means 😆